packages/js/extend-cart-checkout-block/index.js

```diff
@@ -4,7 +4,7 @@ module.exports = {
 		npmDevDependencies: [
 			'@woocommerce/dependency-extraction-webpack-plugin',
 			'@woocommerce/eslint-plugin',
-			'@wordpress/prettier-config@2.25.13',
+			'@wordpress/prettier-config@4.15.0',
```

Comment:
@opr: Would it be possible to relocate this dependency to `package.json`?

Reply:
Hi @kalessil, that's not possible because this `index.js` file is used by `@wordpress/create-block` to _create_ the `package.json` in the resulting extension. If we move it to `package.json` here it would not have any effect.

Reply:
I see, thanks for clarifying!